web/admin/src/pages/document/component/ImportDoc/Feishu.tsx

```diff
@@ -712,18 +696,24 @@ const FeishuImport = ({
                   size='small'
                   sx={{ flexShrink: 0, p: 0, m: 0 }}
                   id='feishu-cloud-select-checkbox'
-                  disabled={items.filter(it => it.space_id === '').length === 0}
+                  disabled={
+                    items.filter(it => it.space_id === 'cloud_disk').length ===
+                    0
+                  }
                   checked={
-                    items.filter(it => it.space_id === '').length > 0 &&
+                    items.filter(it => it.space_id === 'cloud_disk').length >
+                      0 &&
                     items
-                      .filter(it => it.space_id === '')
-                      .every(it => selectIds.includes(it.url))
+                      .filter(it => it.space_id === 'cloud_disk')
+                      .every(it => selectIds.includes(it.doc_id))
                   }
                   onChange={() => {
-                    const spaceItems = items.filter(it => it.space_id === '');
-                    const spaceUrls = spaceItems.map(it => it.url);
+                    const spaceItems = items.filter(
```

Comment:
在Feishu.tsx文件中，部分代码逻辑可以进一步优化以提高可读性和维护性。

```suggestion
// 建议将重复的filter逻辑提取为函数
const filterItemsBySpaceId = (items, spaceId) => 
  items.filter(it => it.space_id === spaceId);

// 使用示例
const spaceItems = filterItemsBySpaceId(items, 'cloud_disk');
const spaceUrls = spaceItems.map(it => it.doc_id);